client.py

```diff
@@ -0,0 +1,20 @@
+import socket
+import sys
+
+from kvstore.constants import INPUT_PROMPT
+
+def main():
+    while True:
```

Comment:
i don't think this loop is handling keyboard interrupts properly. i've seen edge cases where when i shut things down, there remains on open process on my port for a while

Reply:
Yeah you need to catch the interrupt and use that an opportunity to close the socket. By default the OS will keep the socket open for some time after the process has ended. In Python you could do this with a context handler, or explicitly.